appium/tests/specs/inbox/ReadKeyMismatchEmail.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  InboxScreen,
+  EmailScreen
+} from '../../screenobjects/all-screens';
+
+import { CommonData } from '../../data';
+
+describe('INBOX: ', () => {
+
+  it('user is able to view key mismatch email', async () => {
+
+    const senderEmail = CommonData.account.email;
+    const emailSubject = CommonData.keyMismatchEmail.subject;
+    const emailText = CommonData.keyMismatchEmail.message;
+
+    await SplashScreen.login();
+    await SetupKeyScreen.setPassPhrase();
+
+    await InboxScreen.clickOnEmailBySubject(emailSubject);
+    await EmailScreen.checkOpenedEmail(senderEmail, emailSubject, emailText);
```

Comment:
For some reason this check does not work for "key mismatch" screen. Fails on checking sender's email. I'm not so experienced in UI testing to tell why

Reply:
Please see if you can manage to debug it. See if adding accessibility identifier, and querying by that, will help.

Reply:
Also see if adding a delay in front of the check, maybe 100ms, will help.

Reply:
Unfortunately I'm not so good in appium API and JavaScript to do this. I tried to use `accessoryId` but without any success. Test still failing.